packages/modules/src/utils/Types.ts

```diff
@@ -16,23 +17,40 @@ export interface ECDSAOwnershipValidationModuleConfig extends BaseValidationModu
 export interface SessionKeyManagerModuleConfig extends BaseValidationModuleConfig {
   moduleAddress?: string
   version?: ModuleVersion
-  sessionSigner?: Signer
-  sessionPubKey?: string
+  // Review
+  // sessionSigner?: Signer
```

Comment:
this was only meant to be there if we have global public key for all kinds of session leafs generated for this SA+module